.github/BOTMETA.yml

```diff
@@ -3518,10 +3518,6 @@ files:
     supershipit: $team_google
     maintainers: $team_google
     migrated_to: google.cloud
-  $plugins/doc_fragments/grafana.py:
```

Comment:
Any reason why we are removing this?

Reply:
This file was added only to the collection. It never existed in devel.

Reply:
ohk Thanks for the information.